### PR TITLE
Add pyk implication tests to the rpc integration test suite

### PR DESCRIPTION
**Pull request by goodlyrottenapple**
This is the first step in bringing the booster implication endpoint in line with kore. Most of the `.booster-dev` responses will eventually be the same/equivalent to the kore responses.

### CODE REVIEW

File: booster/test/rpc-integration/resources/implies.kore [hunk 1]
```diff
@@ -0,0 +1,2685 @@
+[topCellInitializer{}(LblinitGeneratedTopCell{}()), org'Stop'kframework'Stop'attributes'Stop'Source{}("Source(/private/tmp/pytest-of-sam/pytest-8/popen-gw1/kompiled0/k-files/definition-1.k)")]
```
Comment (jberthold) [inline booster/test/rpc-integration/resources/implies.kore:1]: Maybe we should check in the original K file for this.
Ditto for `implies2.{k,kore}`

File: booster/test/rpc-integration/test-implies2/response-consequent-constraint.booster-dev [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "jsonrpc": "2.0",
+    "id": "4567908768-001",
+    "error": {
+        "code": 6,
+        "data": "unknown constrains",
```
Comment (jberthold) [inline booster/test/rpc-integration/test-implies2/response-consequent-constraint.booster-dev:6]: `s/constrains/constraints/` somewhere in the code...

File: booster/test/rpc-integration/test-implies2/state-antecedent-bottom.send [hunk 3]
```diff
@@ -0,0 +1 @@
...sorts": [{"tag": "SortApp", "name": "SortPgm", "args": []}, {"tag": "SortApp", "name": "SortKItem", "args": []}], "args": [{"tag": "App", "name": "Lblint'UndsSClnUnds'", "sorts": [], "args": [{"tag": "App", "name": "Lbl'UndsCommUnds'", "sorts": [], "args": [{"tag": "DV", "sort": {"tag": "SortApp", "name": "SortId", "args": []}, "value": "$n"}, {"tag": "App", "name": "Lbl'UndsCommUnds'", "sorts": [], "args": [{"tag": "DV", "sort": {"tag": "SortApp", "name": "SortId", "args": []}, "value": "$s"}, {"tag": "App", "name": "Lbl'Stop'List'LBraQuotUndsCommUndsQuotRBra'", "sorts": [], "args": []}]}]}, {"tag": "App", "name": "Lbl'UndsEqlsUndsSCln'", "sorts": [], "args": [{"tag": "DV", "sort": {"tag": "SortApp", "name": "SortId", "args": []}, "value": "$n"}, {"tag": "EVar", "name": "VarY", "sort": {"tag": "SortApp", "name": "SortAExp", "args": []}}]}]}]}, {"tag": "App", "name": "dotk", "sorts": [], "args": []}]}]}, {"tag": "App", "name": "Lbl'-LT-'state'-GT-'", "sorts": [], "args": [{"tag": "App", "name": "Lbl'Stop'Map", "sorts": [], "args": []}]}]}, {"tag": "EVar", "name": "VarGENERATED'Unds'COUNTER'Unds'CELL", "sort": {"tag": "SortApp", "name": "SortGeneratedCounterCell", "args": []}}]}}}}}
```
Comment (jberthold) [inline booster/test/rpc-integration/test-implies2/state-antecedent-bottom.send:1]: This is quite hard to read, maybe we should check in pretty-printed json?
Or, better, add a `README.md` file to the directory that contains a short description of what the data is?
For instance, this one here is not an obvious `Bottom` in the antecedent, what makes it `Bottom` is `X =Int 3` and `X <Int 3` in the constraints.

**Review by jberthold — APPROVED** [review]
Pre-approving. We should add some more human-readable information.

File: booster/test/rpc-integration/resources/implies.kore (hunk 1 above)
Comment (goodlyrottenapple) [reply booster/test/rpc-integration/resources/implies.kore:1]: These are generated via pyk, so not sure how to go about extracting a K definition

**Pull request merged**